libs/SalesforceReact/src/com/salesforce/androidsdk/reactnative/app/SalesforceReactSDKManager.java

```diff
@@ -139,27 +147,31 @@ public List<NativeModule> createNativeModules(
 				return modules;
 			}
 
+			/** @noinspection unused*/
 			public List<Class<? extends JavaScriptModule>> createJSModules() {
 				return Collections.emptyList();
 			}
 
+			@NonNull
 			@Override
-			public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
+			public List<ViewManager> createViewManagers(
+					@NonNull ReactApplicationContext reactContext
+			) {
 				return Collections.emptyList();
 			}
 		};
 	}
 
+	@NonNull
 	@Override
-	protected LinkedHashMap<String, DevActionHandler> getDevActions(final Activity frontActivity) {
-		LinkedHashMap<String, DevActionHandler> devActions = super.getDevActions(frontActivity);
+	protected Map<String, DevActionHandler> getDevActions(
```

Comment:
Everything in this source is inspector cleanup for existing trivial issues and those introduced by the Kotlin conversion.  The one exception is this change from a Collections implementation to an Interface type, which is downstream of a Kotlin conversion and best practice anyways.